### PR TITLE
Update protob

### PR DESCRIPTION
**Pull request by therealssj**
Changes:
- Update protob to master
- Add travis check for protob commit hash 
- Add `verify_protob_hash` Make target

Does this change need to mentioned in CHANGELOG.md?
no

Requires changes in protobuff specs?
no

Requires testing
no

### CODE REVIEW

File: ci-scripts/verify_protob_hash.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# Get commit hash from submodule
+cd tiny-firmware/protob/
```
Comment (olemis) [inline ci-scripts/verify_protob_hash.sh]: `git -C ...`

File: ci-scripts/verify_protob_hash.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+PROTOB_HASH=$(git log --pretty=oneline | head -1 | cut -c 1-40)
```
Comment (olemis) [inline ci-scripts/verify_protob_hash.sh]: `git -C ... rev-parse ...` ?

File: ci-scripts/verify_protob_hash.sh [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```
Comment (olemis) [inline ci-scripts/verify_protob_hash.sh]: could this be a `Makefile` target ?

File: ci-scripts/verify_protob_hash.sh [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+# Get commit hash from submodule
+cd tiny-firmware/protob/
+PROTOB_HASH=$(git log --pretty=oneline | head -1 | cut -c 1-40)
+
+# clone protob repository from git
+mkdir -p tmp/hardware-wallet-protob
+git clone --depth=50 --single-branch --branch master https://github.com/skycoin/hardware-wallet-protob.git tmp/hardware-wallet-protob
```
Comment (olemis) [inline ci-scripts/verify_protob_hash.sh]: No need to clone repository e.g. [use `git ls-remote`](https://stackoverflow.com/questions/24750215/getting-the-last-commit-hash-from-a-remote-repo-without-cloning).

Comment (therealssj) [reply ci-scripts/verify_protob_hash.sh]: `ls-remote` doesn't let me look at old commits.
its limited to refs only.
I am checking 50 old commits to see if the submodule revision hash is in them or not.

Comment (olemis) [reply ci-scripts/verify_protob_hash.sh]: use github api e.g. https://api.github.com/repos/skycoin/hardware-wallet/commits?per_page=50&sha=develop

Comment (olemis) [reply ci-scripts/verify_protob_hash.sh]: ... or `git fetch` into the same repository ?

Comment (olemis) [reply ci-scripts/verify_protob_hash.sh]: ... or `git fetch` into the same repository ? 

> use github api

see [SO 24344835](https://stackoverflow.com/questions/24344835/using-tilde-or-caret-with-git-ls-remote/55805039#55805039)

Comment (therealssj) [reply ci-scripts/verify_protob_hash.sh]: done

File: ci-scripts/verify_protob_hash.sh (hunk 3 above)
Comment (therealssj) [reply ci-scripts/verify_protob_hash.sh]: done

**Pull request merged**